style.css

```diff
@@ -0,0 +1,479 @@
+@import url('http://fonts.cdnfonts.com/css/cocogoose');
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap');
+
+body {
+  margin: 0;
+  padding: 0;
+  text-align: center;
+}
+
+.dNav {
+  transition: 300ms ease-in-out;
+  width: 90%;
+  position: absolute;
+  padding: 5%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  transform: translateY(-94%);
+  background-color: rgb(223, 215, 215);
+  color: #272a31;
+  box-shadow: 0 3px 3px 0 rgba(0, 0, 0, 0.75);
+  -webkit-box-shadow: 0 3px 3px 0 rgba(0, 0, 0, 0.75);
+  -moz-box-shadow: 0 3px 3px 0 rgba(0, 0, 0, 0.75);
+}
+
+.brand {
+  display: none;
+}
+
+.Campaign {
+  display: none;
+}
+
+.partners {
+  display: none;
+}
+
+.card {
+  display: flex;
+  background-color: #5f5f5f50;
+  flex-direction: row;
+  gap: 8.5px;
+  align-items: center;
+}
+
+.card:hover {
+  transform: scale(1.1);
+}
+
+.join {
+  background-color: #ec5242;
+  padding: 5%;
+  font-weight: 700;
+  margin: 27px;
+}
+
+.join:hover {
+  background-color: rgb(240, 113, 113);
+}
+
+ul {
+  margin-top: 37px;
+  list-style: none;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 6px;
+  margin-bottom: 22px;
+}
+
+.dNav a {
+  text-decoration: none;
+  color: #272a31;
+}
+
+.hamburger {
+  background-color: rgb(223, 215, 215);
+  border: none;
+  position: relative;
+  top: 10px;
+  right: 42%;
+  z-index: 10;
+  cursor: pointer;
+}
+
+.hamburger span {
+  transition: 300ms ease-in-out;
+  background-color: #272a31;
+  display: block;
+  width: 1.6rem;
+  height: 3px;
+  margin: 3.5px;
+  font-weight: 900;
+  border-radius: 10px;
+}
+
+.intro :nth-child(4) {
+  font-weight: bold;
+  font-size: 1.7em;
+}
+
+.fcontent :nth-child(1) {
+  display: none;
+}
+
+.active.hamburger :nth-child(2) {
+  opacity: 0;
+}
+
+.active.hamburger :nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+}
+
+.active.hamburger :nth-child(3) {
+  transform: translateY(-4.5px) rotate(-45deg);
+}
+
+.dNav.active {
+  transform: translateY(-15%);
+}
+
+.dNav li:hover {
+  background-color: grey;
+}
+
+.intro {
+  padding-top: 10%;
+  padding-left: 17px;
+  padding-right: 17px;
+  padding-bottom: 17px;
+  background: url(./assest/event.png);
+  background-size: cover;
+  text-align: left;
+  background-repeat: no-repeat;
+  font-family: 'cocogoose', sans-serif;
+}
+
+.description {
+  background-color: rgba(255, 255, 255, 0.616);
+  font-family: 'lato', sans-serif;
+  font-weight: 500;
+  border: 2px solid white;
+}
+
+.see-more {
+  display: none;
+}
+
+.hello {
+  margin-bottom: 0;
+  font-size: 1.7em;
+}
+
+.intro .hello,
+.intro h1,
+.card span {
+  color: #ec5242;
+}
+
+.mainProgram {
+  padding: 3%;
+  background-color: #272a31;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 8px;
+}
```

Comment:
- [ ] For this section the background is looking okay but you can improve it to look like the design check images below 👇🏻  🤔  I think you don't need to use the `background-color: #272a31;` instead you can use an image with a `background-repeat: repeat;` property. What do you think?

- [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07) design:

![og](https://user-images.githubusercontent.com/44745855/132837558-b3e87012-6b81-43f7-9b74-dd6a88f8adab.png)

- Your design:

![30main](https://user-images.githubusercontent.com/79058364/176779476-db1f6765-61de-43b3-bc5c-56fc8cf7b2c5.png)

Reply:
Hello @BenMukebo,
Thanks for your effort your review is amazing it includes every single detail…
But to this one 
Isn’t this a personal preference ?
I mean there is a hover effect,
But dose it have to be the one you pick ?